src/main/java/com/skhu/luxuryshop/user/dto/UserSignupDto.java

```diff
@@ -0,0 +1,48 @@
+package com.skhu.luxuryshop.user.dto;
+
+import com.skhu.luxuryshop.user.entity.UserEntity;
+import com.skhu.luxuryshop.user.exception.SignupPasswordUnmatchedException;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import org.hibernate.validator.constraints.Length;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
+
+@Getter
+@AllArgsConstructor
+public class UserSignupDto {
+    @NotNull
+    @Email
+    private String email;
+
+    @NotNull
+    @Length(min = 8, max = 20)
+    private String password;
+
+    @NotNull
+    @Length(min = 8, max = 20)
+    private String passwordCheck;
+
+    @NotNull
+    @Length(min = 2, max = 8)
+    private String nickname;
+
+    public UserEntity toUserEntity() {
+        isSamePassword(password, passwordCheck);
+        UserEntity user = UserEntity.builder()
+                .email(email)
+                .password(password)
+                .nickname(nickname)
+                .build();
+        return user;
+    }
+
+    //비밀번호 일치 불일치 검사
+    public boolean isSamePassword(String password, String passwordCheck) {
+        if (password.equals(passwordCheck))
+            return true;
+        throw new SignupPasswordUnmatchedException("비밀번호 불일치");
```

Comment:
네이밍이 isSamePassword 이면 리턴타입이 boolean으로 예상되고 당연히 if문 안에 있을 것 같은데
맨 윗줄에 덩그라니 있으니 조금 부자연스러워 보여요!
이런식으로 작성하는건 어떨까요!??
```suggestion
    public UserEntity toUserEntity() {
        validateSmaePassword(password, passwordCheck);
        UserEntity user = UserEntity.builder()
                .email(email)
                .password(password)
                .nickname(nickname)
                .build();
        return user;
    }

    public void validateSmaePassword(String password, String passwordCheck) {
        if (!password.equals(passwordCheck)) {
            throw new SignupPasswordUnmatchedException("비밀번호 불일치");
        }
    }
```
이런식으로 검증했을 것 같아요!
그리고 if문은 내용이 한 줄이라도 중괄호를 쳐주는게 주로 사용하는 컨벤션이더라구용.